framework/src/utils/RandomGenerator.C

```diff
@@ -0,0 +1,45 @@
+/****************************************************************/
+/*               DO NOT MODIFY THIS HEADER                      */
+/* MOOSE - Multiphysics Object Oriented Simulation Environment  */
+/*                                                              */
+/*           (c) 2010 Battelle Energy Alliance, LLC             */
+/*                   ALL RIGHTS RESERVED                        */
+/*                                                              */
+/*          Prepared by Battelle Energy Alliance, LLC           */
+/*            Under Contract No. DE-AC07-05ID14517              */
+/*            With the U. S. Department of Energy               */
+/*                                                              */
+/*            See COPYRIGHT for full restrictions               */
+/****************************************************************/
+
+#include "RandomGenerator.h"
+#include  <random>
+
+class MersenneTwister
+{
+public:
+  std::mt19937 _mt_eng;
+};
```

Comment:
It seems c++11 Random modules include most of functions inside the external library "randistrs.h". Do you want to keep using the external library or the do you want to switch to c++11 Random modules?